models/concerns/validate_data.rb

```diff
@@ -139,6 +139,14 @@ def check_services(schedule)
   def check_relation_consistent_timewindows
     unconsistent_relation_timewindows = []
     @hash[:relations].each{ |relation|
+      unless relation[:linked_ids].uniq == relation[:linked_ids]
+        raise OptimizerWrapper::DiscordantProblemError.new('Same service ID was provided several times in one relation.')
+      end
+
+      unless relation[:linked_vehicle_ids].uniq == relation[:linked_vehicle_ids]
+        raise OptimizerWrapper::DiscordantProblemError.new('Same vehicle ID was provided several times in one relation.')
```

Comment:
There is a subtle difference between several and multiple. We want "multiple" because it means "more than once" -- or we can directly use 'more than once' in both sentences.

Same vehicle ID is provided multiple times in the same relation.
Same vehicle ID is provided in the same relation more than once.